source/changelog.txt

```diff
@@ -12,6 +12,31 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v1.10.3
+-------
+
+*Released July 31, 2023*
+
+Updates environment variables related to telemetry.
+
+`Full release notes available on JIRA
+<https://jira.mongodb.org/browse/MONGOSH-1545?jql=project%20%3D%20MONGOSH%20AND%20fixVersion%20%3D%201.10.3>`__.
```

Comment:
Question: Not sure why, but this Jira page is very different from those on the other release notes.  It looks more like what we'd expect in the Server docs for a Jira link.  Is this an intentional change?

Reply:
This is a great question. Thanks @kennethdyer. I'm going to leave this open for the tech review. I couldn't find the typical jira based Release Notes for the 1.10.3 release. This might be because of the nature of the release, but I thought I'd ask @addaleax about this in external review as well.

Reply:
This would be the "standard format" release notes link: https://jira.mongodb.org/secure/ReleaseNote.jspa?projectId=17381&version=36452

as you can see, it's not particularly meaningful, but you can still feel free to use it here for consistency. If not, I'd just omit it.

Reply:
Thanks @addaleax! I think I will add it for consistency. I appreciate the quick feedback!